turbo.json

```diff
@@ -7,7 +7,7 @@
     },
     "test": {
       "outputs": ["coverage/**"],
-      "dependsOn": []
+      "dependsOn": ["^build"]
```

Comment:
required now since we rely on a build version of our design tokens when running the tests